### PR TITLE
Improvements V1

### PR DESCRIPTION
**Pull request by gh03ty**
Issues solved:
#3 #4 #5 #6 #7

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,21 +2,44 @@
...
 
-var arguments = process.argv.slice(2);
+let arguments = process.argv.slice(2);
 
-var operand1 = arguments[0];
-var operand2 = arguments[1];
-var operator = arguments[2];
+/* FIX #4 - Calculation is not possible with Strings */
+let operand1 = parseInt(arguments[0], 10);
+let operand2 = parseInt(arguments[1], 10);
```
Comment (sschzrk) [inline index.js:8]: Why ParseInt(x,10)
No calc with decimal numbers

**Review by sschzrk — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -2,21 +2,44 @@
  * Reverse Polish Notation Calculator
  */
 
-var arguments = process.argv.slice(2);
+let arguments = process.argv.slice(2);
 
-var operand1 = arguments[0];
-var operand2 = arguments[1];
-var operator = arguments[2];
+/* FIX #4 - Calculation is not possible with Strings */
```
Comment (sschzrk) [inline index.js]: Nice coms to doc fix(s)

File: index.js [hunk 3]
```diff
@@ -2,21 +2,44 @@
...
+    break;
+  case 'p':
+    result = Math.pow(operand1, operand2);
+    break;
+  /*                                                        */
   default:
-    console.error('Not implemented:', operator);
+    console.error('Not implemented or Is not a mathamatical operator :', operator);
+    /* FIX #5 - Additional help for userfriendliness added           */
+    console.log('Supported operators are: + , - , / , * , p (power).');
```
Comment (sschzrk) [inline index.js]: Nice error message to make it user friendly

File: index.js [hunk 4]
```diff
@@ -2,21 +2,44 @@
...
+    break;
+  case '*':
+    result = operand1 * operand2;
+    break;
+  case '/':
+    result = operand1 / operand2;
+    break;
+  case 'p':
+    result = Math.pow(operand1, operand2);
+    break;
```
Comment (sschzrk) [inline index.js]: cool operator

File: README.md [hunk 5]
```diff
@@ -1 +1,16 @@
-# Calc
\ No newline at end of file
+# Reverse Polish Notation Calculator 
+
+Die umgekehrte polnische Notation (UPN) oder reverse polnische Notation englisch reverse Polish notation (kurz RPN), auch Postfixnotation genannt, ist eine von der polnischen Notation abgeleitete Schreibweise bzw. Eingabelogik für die Anwendung von Operationen. Bei der umgekehrten polnischen Notation werden zunächst die Operanden niedergeschrieben bzw. eingegeben und danach der darauf anzuwendende Operator.
+
+### Anforderungen
+
+Zum Ausführen dieses Javascripts wird *NodeJS* benötigt !
```
Comment (ShinyKampfkeule) [inline README.md]: Please explain how to get NodeJS ! I´m a NOOB and Google doesn´t tell me ! xC

File: index.js (hunk 1 above)
Comment (dwdab) [reply index.js:8]: @FearZee Please create a dedicated issue for that.

**Review by sschzrk — APPROVED** [review]

**Pull request merged**